packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -166,17 +167,34 @@ export default function useBlockDropZone( {
 		[ targetRootClientId ]
 	);
 
-	const { getBlockListSettings, getBlocks, getBlockIndex } =
-		useSelect( blockEditorStore );
+	const {
+		getBlockListSettings,
+		getBlocks,
+		getBlockIndex,
+		getDraggedBlocksOrigin,
+	} = useSelect( blockEditorStore );
```

Comment:
```suggestion
	const {
		getBlockListSettings,
		getBlocks,
		getBlockIndex,
		getDraggedBlocksOrigin,
	} = useSelect( unlock( blockEditorStore ) );
```

I tried to make the `getDraggedBlocksOrigin` a private selector using lock/unlock. I tried the idea above but it didn't work. I was using my work in https://github.com/WordPress/gutenberg/pull/47638/as a guide.

Reply:
@adamziel Do you know of a pattern to allow me to do this?

Reply:
@getdave it's `unlock( useSelect(`, not `useSelect( unlock( `

Reply:
✅